src/acsets/acsets.py

```diff
@@ -203,33 +398,91 @@ def set_subpart(self, i: int, f: Property, x: any):
             self._subparts[f][i] = x
 
     def has_subpart(self, i: int, f: Property):
+        """Check if a property exists for a given row in a table of the ACSset.
+
+        Args:
+            i: The row index for the property mapping to be added to.
+            f: The `Hom` or `Attr` to check for.
+
+        Returns:
+            `True` if the property `f` exists on row `i` or `False` if it doesn't.
+        """
         return i in self._subparts[f].keys()
 
     def subpart(self, i: int, f: Property, oneindex=False):
+        """Get the subpart of a part in an ACSet
+
+        Args:
+            oneindex (boolean): Whether or not to return the index starting at 1 or 0, default is `False` which is zero-indexed
+            i: The part that you are indexing.
+            f: The `Hom` or `Attr` to retrieve.
+
+        Returns:
+            The subpart of the ACset.
+        """
         if oneindex and type(f) == Hom:
             return self._subparts[f][i] + 1
         else:
             return self._subparts[f][i]
 
     def nparts(self, ob: Ob) -> int:
+        """Get the number of rows in a given table of the ACSet.
+
+        Args:
```

Comment:
Somehow this is not what I would say. I would say that `ob` is the name of the table, maybe? "The `Ob` table sounds like there is a table named `Ob`

Reply:
I have revised this in the latest commit